torch/quantization/fx/quantize.py

```diff
@@ -487,14 +512,14 @@ def load_arg(a):
                 # parent
                 if qconfig is not None:
                     assert obj is not None
-                    insert_observer_for_special_module(
+                    standalone_module_input_idxs = insert_observer_for_special_module(
```

Comment:
optional readability nit (not related to this PR, and optional): maybe we can call the function `maybe_insert_observer_for_special_module`, or move the gating here?